leonardo-example.conf

```diff
@@ -142,3 +142,13 @@ serviceAccounts {
     providerTimeout = 30 seconds
   }
 }
+
+#Configs to auto stop cluster after idle period
+autoFreeze {
+  #This is the duration after which the data accessed is persisted in the ClusterDateAccessedActor
+  dateAccessedMonitorScheduler = 1 m
```

Comment:
Minor, can you use the full time unit to be consistent with the rest of this file? E.g. `1 minute`, etc.